openmc/mgxs/library.py

```diff
@@ -1410,6 +1420,8 @@ def check_library_for_openmc_mgxs(self):
           fixed source problem could be the target.
         - Fission and kappa-fission are not required as they are only
           needed to support tallies the user may wish to request.
+        - Scattering multiplicity should have been tallied, either using a
+        multiplicity or scatter and nu-scatter matrix tally.
```

Comment:
Also indent "multiplicity" to line up with the "Scattering..." text in the above line